tests/boardfarm_plugins/boardfarm_prplmesh/prplmesh_config.json

```diff
@@ -0,0 +1,52 @@
+{
+    "prplmesh_docker": {
+        "name": "gateway",
+        "board_type": "prplmesh_docker",
+        "role": "controller",
+        "delay": 7,
+        "use_tcpdump": true,
+        "docker_network": "prplmesh-net",
+        "conn_cmd": "",
+        "connection_type": "local_cmd",
+        "devices": [
+            {
+                "name": "repeater1",
+                "type": "prplmesh_docker",
+                "role": "agent",
+                "delay": 7,
+                "use_tcpdump": true,
+                "docker_network": "prplmesh-net",
+                "conn_cmd": "",
+                "radios": [
```

Comment:
I would like to start out with a minimum config. It should have none of the options where we use the default (and conversely, the defaults should be what we need, e.g. delay).

The minimum config for now is just the top-level device without any subdevices.

Overall this needs to be split up into commits that add things in small steps. Now it's difficult to see why something is needed (and if it is really needed).

Reply:
> Overall this needs to be split up into commits that add things in small steps. Now it's difficult to see why something is needed (and if it is really needed).

Applied to #1108.
Added minimal config with empty subdevices list.
(`devices` list should be present even if it is empty)